aedes.js

```diff
@@ -56,9 +53,8 @@ export class Aedes extends EventEmitter {
       return new Client(that, conn, req)
     }
 
-    this._parallel = parallel()
-    this._series = series()
-    this._enqueuers = reusify(DoEnqueues)
+    this._series = runSeries
+    this._enqueuers = new ObjectPool(DoEnqueues)
```

Comment:
It might be interesting to verify if this is needed at all. My 2 cents is that we don't need this at all anymore and there is no perf gap in just allocating them all the time.

Reply:
Just ran the benchmark on the commit that only removes the object pool. No material difference, still about 18% improvement over the main branch. See: https://github.com/moscajs/aedes/actions/runs/17486480074/attempts/1#summary-49666422333